boards/beagle/beagleconnect_freedom/revision.cmake

```diff
@@ -1,5 +1,9 @@
 set(BOARD_REVISIONS "C5" "C7")
 
+if(NOT DEFINED BOARD_REVISION)
+  set(BOARD_REVISION ${LIST_BOARD_REVISION_DEFAULT})
+endif()
+
```

Comment:
```suggestion
```

Reply:
I would at least like to have support for `beagleconnect_freedom/cc1352p5` Requiring `beagleconnect_freedom@C7/cc1352p7` for board name seems rather verbose.